app/dashboard/templates/dashboard/setting.html

```diff
@@ -85,18 +85,32 @@
     </div>
     <!-- END change name & profile picture -->
 
+    <div class="card">
+      <div class="card-body">
+        <div class="card-title">Security Key (WebAuthn)</div>
+        <div class="mb-3">
+          You can secure your account by linking either your FIDO-supported physical key such as Yubikey, Google Titan,
+          or a device with appropriate hardware to your account.
+        </div>
+        {% if current_user.fido_uuid is none %}
```

Comment:
This check can be replaced by `{% if not current_user.fido_uuid %}` I think

Reply:
The reason that I am checking if it's `None` is to emphasize its type. `fido_uuid` could possibly be `None` or `str`, using `not` will allow some unintended behavior like if I set `fido_uuid` to `''` this is also a `False` but not `None`.